crates/router/src/connector/adyen/transformers.rs

```diff
@@ -293,6 +298,7 @@ pub struct Amount {
 
 #[derive(Debug, Clone, Serialize)]
 #[serde(tag = "type")]
+#[serde(rename_all = "snake_case")]
```

Comment:
This will have impact on all the payment method, Do we need it?

Reply:
removed ``#[serde(rename_all = "snake_case")]``